web/app.js

```diff
@@ -717,7 +717,7 @@ const PDFViewerApplication = {
     }
 
     if (this.supportsIntegratedFind) {
-      appConfig.toolbar?.viewFind?.classList.add("hidden");
+      appConfig.findBar?.toggleButton?.classList.add("hidden");
```

Comment:
I have tested this code path by setting `supportsIntegratedFind` to `true` and noticing that the find bar toggle button still disappears like before.